lib/index.js

```diff
@@ -8,6 +8,7 @@ var Spawn = require("child_process").spawn
   , Ul = require("ul")
   , OneByOne = require("one-by-one")
   , Sliced = require("sliced")
+  , Q = require('q')
```

Comment:
> I am still pretty fresh in nodejs ecosystem and literally yesterday I learned about libs like promisify - I think that this is how it should work :D

Oh, nice work, man! :facepunch: 

Generally, I try to maintain modules as small as possible–so, like I specified before, I am for creating a `cobol-promises` project. I can create it, if you like. :sparkler: